CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## [v2.0.1](https://github.com/singer-io/tap-jira/tree/v2.0.1) (2021-04-06)
+
+[Full Changelog](https://github.com/singer-io/tap-jira/compare/v2.0.0...v2.0.1)
+
+**Closed issues:**
+
+- Status names mapping to status subcategories [\#50](https://github.com/singer-io/tap-jira/issues/50)
+- primary key of Users stream needs to be updated [\#48](https://github.com/singer-io/tap-jira/issues/48)
```

Comment:
I'd remove these closed issues, they were opened and closed by the submitters, not for code changes

Reply:
Sure

Reply:
Done